modules/books.js

```diff
@@ -0,0 +1,33 @@
+export default class BookCollection {
+  constructor() {
+    this.booksArray = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  addBook = (title, author) => {
+    const book = {
+      title,
+      author,
+    };
+    this.booksArray.push(book);
+    localStorage.setItem('books', JSON.stringify(this.booksArray));
+    this.displayBooks();
+  }
+
+  removeBook = (index) => {
```

Comment:
- [ ] After many tries, it's impossible for me to remove the very first book on the list.  The remove button is only working for me in all books but not on the first one.  Tested both on Firefox and Chrome.

Please, kindly correct this bug so the app works as intended. 💪

Reply:
@Meltrust thank you for pointing out the bug, turned out that when you create the event listeners while creating the HTML elements all inside a loop, the previous event listeners are lost. I fixed it 😄